calico/reference/resources/felixconfig.md

```diff
@@ -82,7 +82,8 @@ spec:
 | reportingInterval                  | Interval at which Felix reports its status into the datastore, or 0 to disable.  Must be non-zero in OpenStack deployments. | `5s`, `10s`, `1m` etc. | duration | `30s` |
 | reportingTTL                       | Time-to-live setting for process-wide status reports. | `5s`, `10s`, `1m` etc. | duration | `90s` |
 | routeRefreshInterval               | Period at which Felix re-checks the routes in the dataplane to ensure that no other process has accidentally broken {{site.prodname}}'s rules. Set to 0 to disable route refresh. | `5s`, `10s`, `1m` etc. | duration | `90s` |
-| routeTableRange                    | Calico programs additional Linux route tables for various purposes.  `RouteTableRange` specifies the indices of the route tables that Calico should use. |  | [RouteTableRange](#routetablerange) | `{Min: 1, Max: 250}` |
+| routeTableRange                    | *deprecated in favour of `RouteTableRanges`* Calico programs additional Linux route tables for various purposes. `RouteTableRange` specifies the indices of the route tables that Calico should use. If explicitly-set, will be honoured in favour of `RouteTableRanges`. |  | [RouteTableRanges](#routetablerange) | `{Min: 1, Max: 250}` |
```

Comment:
Left the default on this one.

Reply:
good catch, thought I got them all :laughing:

Reply:
Warning: American incoming

"honoured -> honored"

"favour" -> "favor" 

alternatively, we can avoid those contentious words altogether :laughing: 

> If explicitly set, this field will take precendence over RouteTableRanges

Non-american comment: "explicitly-set" shouldn't be hyphenated. "explicitly set" is fine.

Reply:
s/precendence/precedence

Reply:
cleaned up this wording now, although worth noting I saw a few unrelated parts of our codebase that still use european spelling, I dont know if a formal decision has been made around stuff like this (?), but maybe another, more intentional PR could grab those remaining bits if we want to make a concerted effort there